health/micro-ui-react19/web/packages/css/src/pages/employee/campaign.scss

```
@@ -1260,4 +1261,126 @@
 .digit-popup-footer{
   border-bottom-left-radius: 12px;
   border-bottom-right-radius: 12px;
+}
+.reports-configuration {
+  &__heading {
+  font-family: var(--digitv2-fontFamily-rc);
+  font-weight: var(--digitv2-fontWeight-bold);
+  font-style: Bold;
```

Stylelint (17.7.0)

[error] Expected "Bold" to be "bold" (value-keyword-case)

(value-keyword-case)